Flow.Launcher/Helper/SingleInstance.cs

```diff
@@ -400,6 +400,7 @@ private class IPCRemoteService : MarshalByRefObject
             /// to ensure that lease never expires.
             /// </summary>
             /// <returns>Always null.</returns>
+            [Obsolete] // overrides an obsolete method, so also needs to be obsolete
```

Comment:
I am unsure why we mark this as Obsolete. I think this is not likely to be a public access api.

Reply:
It's actually completely obsolete (i.e. will throw if called): https://docs.microsoft.com/en-us/dotnet/api/system.marshalbyrefobject.initializelifetimeservice?view=net-6.0

I've removed the whole thing as I don't see any references to it anywhere.

Reply:
I am curious why do we have it before. The file is used for maintaining only one instance.

Reply:
Here's a good reference to it: https://github.com/Flow-Launcher/Flow.Launcher/commit/be33ac3c4f29e4d71b6684fd76e087dd076c50a4

It used to have multiple usages within https://github.com/Flow-Launcher/Flow.Launcher/blob/be33ac3c4f29e4d71b6684fd76e087dd076c50a4/Wox/Helper/SingleInstance.cs (see below), but these were removed in https://github.com/Flow-Launcher/Flow.Launcher/commit/d329b73748861a0cad7844897285c83bbe36f9d3#diff-5e380b0d6c890e4b05cd9c291ce4618cdfdf21b1cb5c5458ddc7bd25f97d7706 which only did a partial cleanup (should have removed this class at that time).

Original usage: https://github.com/Flow-Launcher/Flow.Launcher/blob/be33ac3c4f29e4d71b6684fd76e087dd076c50a4/Wox/Helper/SingleInstance.cs#L388-L410

Reply:
That's super nice. Thank you!